app/view_utils/path_helpers.rb

```diff
@@ -56,7 +56,7 @@ def landing_page_path(community_id:, logged_in:, locale_param:, default_locale:)
   end
 
   def paths
-    RequestStore.store[:url_helpers] ||= Rails.application.routes.url_helpers
+    @_url_helpers ||= Rails.application.routes.url_helpers
```

Comment:
`@_url_helpers` in `path_helpers` won't be scoped to a request. Is this as intended?